README.md

```diff
@@ -1,220 +1,174 @@
-| [deployment-namespace-rbac.yml](https://raw.githubusercontent.com/hawtio/hawtio-online/master/deployment-namespace-rbac.yml) | Same as `deployment-namespace-os4.yml`, with configurable RBAC for Jolokia requests authorization. See the [RBAC](#rbac) section for more information. |
-| [deployment-cluster-rbac.yml](https://raw.githubusercontent.com/hawtio/hawtio-online/master/deployment-cluster-rbac.yml) | Same as `deployment-cluster-os4.yml`, with configurable RBAC for Jolokia requests authorization. See the [RBAC](#rbac) section for more information. |
+#### Proxying certificates
+
+For OpenShift, a client certificate must be generated using the [service signing certificate][service-signing-certificate] authority private key.
 
 [service-signing-certificate]: https://docs.openshift.com/container-platform/latest/security/certificates/service-serving-certificate.html
 
-To deploy the Hawtio Online console, execute the following command:
+Run the following script to generate and set up a client certificate for Hawtio Online:
 
 ```sh
-$ oc new-app -f https://raw.githubusercontent.com/hawtio/hawtio-online/master/deployment-namespace.yml \
-  -p ROUTE_HOSTNAME=<HOST>
+$ ./scripts/generate-proxying.sh
 ```
 
-Note that the `ROUTE_HOSTNAME` parameter can be omitted when using the `deployment-namespace` template.
-In that case, OpenShift automatically generates one for you.
-
-You can obtain more information about the template parameters, by executing the following command:
+or if you have Yarn installed, this will also do the same thing:
 
 ```sh
-$ oc process --parameters -f https://raw.githubusercontent.com/hawtio/hawtio-online/master/deployment-namespace.yml
-NAME                DESCRIPTION                                                                   GENERATOR           VALUE
-ROUTE_HOSTNAME      The externally-reachable host name that routes to the Hawtio Online service
+$ yarn gen:proxying
 ```
 
-You can obtain the status of your deployment, by running:
+#### Serving certificates
 
-```sh
-$ oc status
-In project hawtio on server https://192.168.64.12:8443
+For OpenShift, a serving certificate is automatically generated for your Hawtio Online deployment using the [service signing certificate][service-signing-certificate] feature.
 
-https://hawtio-online-hawtio.192.168.64.12.nip.io (redirects) (svc/hawtio-online)
-  dc/hawtio-online deploys istag/hawtio-online:latest 
-    deployment #1 deployed 2 minutes ago - 1 pod
-```
+### Kubernetes
 
-Open the route URL displayed above from your Web browser to access the Hawtio Online console.
+#### Proxying certificates
 
-## OpenShift 4
+For Kubernetes, proxing certificates are not supported, so you don't need to go through the steps.
```

Comment:
Should we rather say it's disabled by default. It's possible to use a proxying client certificate, even for Kubernetes, the may difference with OpenShift is that a custom CA must be generated / used, and mounted / configured into the Jolokia agent.

Reply:
Thanks. Yes, you are right. I'll change that later.

Reply:
Agreed, that can be improvement in a subsequent PR.